app/views/groups/_update.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= form_with(model: group) do |form| %>
```

Comment:
This file is mostly the same as `_form`. Seems you want to change the button name?
If so, this[ thread](https://stackoverflow.com/questions/35815810/suplying-different-names-for-submit-buttons-in-forms) may help.

Reply:
Thank you Ming, when I added the _update partial I immediately thought about the DRY principle haha, and had the feeling there had to be a different way to display the name of the buttons. Thank you for suggesting this article!